.travis.yml

```diff
@@ -7,6 +7,9 @@ notifications:
   email: false
 node_js:
   - '8'
+  - '6'
+  - '4'
+before_install: if [[ `npm -v` < 3* ]]; then npm i -g npm@^3; fi
```

Comment:
I disagree. There's a difference between what we support and what we test with. I'm 99.99% certain that is using npm5 will not result in us missing a bug that we could otherwise catch if we used npm3